src/Core/src/ImageSources/UriImageSourceService/UriImageSourceService.iOS.cs

```diff
@@ -1,16 +1,18 @@
 ﻿#nullable enable
 using System;
+using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
 using Foundation;
 using Microsoft.Extensions.Logging;
-using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui
 {
 	public partial class UriImageSourceService
 	{
+		internal string CacheDirectory = Path.Combine(Essentials.FileSystem.CacheDirectory, "com.microsoft.maui.sample", "MauiUriImages");
```

Comment:
I don't know the context behind this name, but do we want to keep the `sample` in the path?

Reply:
Could be `cache` instead of `sample` and in the future use this path as a root directory for other caches (?)

Reply:
updated, thanks for the feedback!